x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_regression.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { FeatureImportance, TopClasses } from '../../../../../common/types/feature_importance';
+import { findMaxMin, useDecisionPathData } from './use_classification_path_data';
+import { DecisionPathChart } from './decision_path_chart';
+
+interface RegressionDecisionPathProps {
+  predictionFieldName?: string;
+  baseline?: number;
+  predictedValue?: number | undefined;
+  featureImportance: FeatureImportance[];
+  topClasses?: TopClasses;
+}
+
+export const RegressionDecisionPath: FC<RegressionDecisionPathProps> = ({
+  baseline,
+  featureImportance,
+  predictedValue,
+  predictionFieldName,
+}) => {
+  const { decisionPathData } = useDecisionPathData({
+    baseline,
+    featureImportance,
+    predictedValue,
+  });
+  const domain = useMemo(() => {
+    let maxDomain;
+    let minDomain;
+    // if decisionPathData has calculated cumulative path
+    if (
+      Array.isArray(decisionPathData) &&
+      decisionPathData.length > 0 &&
+      decisionPathData[0].length === 3
+    ) {
+      const { max, min } = findMaxMin(decisionPathData, (d: [string, number, number]) => d[2]);
+      maxDomain = max;
+      minDomain = min;
+      const buffer = Math.abs(maxDomain - minDomain) * 0.1;
+      maxDomain =
+        (typeof baseline === 'number' ? Math.max(maxDomain, baseline) : maxDomain) + buffer;
+      minDomain =
+        (typeof baseline === 'number' ? Math.min(minDomain, baseline) : minDomain) - buffer;
+    }
+    return { maxDomain, minDomain };
+  }, [decisionPathData, baseline]);
+
+  if (!decisionPathData) return <div />;
```

Comment:
Can you add a comment about reasons we could end up with returning an empty `div` here?

Reply:
Changed it to a EuiCallOut here https://github.com/elastic/kibana/pull/73561/commits/ef815d11a2bf334d703366daff3519b38b848e11